corehq/apps/saved_reports/models.py

```diff
@@ -146,6 +147,7 @@ def by_domain_and_owner(cls, domain, owner_id, report_slug=None,
         return configs
 
     @classmethod
+    @quickcache(['domain', 'only_id', 'stale'], timeout=5)
```

Comment:
I think you want:
```py
@quickcache(['domain', 'only_id'], skip_arg='stale')
```
Is there a reason to set the timeout so low? If you are worried about the cache getting stale, you should explicitly bust it whenever this value is likely to change.

Reply:
@proteusvacuum Why do I want to skip `stale`? Won't the results potentially vary on the value of that argument? (to be fair, it's never really called with any argument other than stale=False)

The timeout was chosen to be valid for at least a (very) long running request, but busting it where it could change is maybe a better way of going about solving performance issues rather than "keeping it dynamic".

Reply:
Oh, you're right!

Reply:
I'm just going to remove the `stale` and hard-code the stale argument to `ReportNotification.by_domain`.